tests/rptest/tests/audit_log_tests.py

```diff
@@ -155,25 +563,378 @@ def combine(acc, x):
+                            "acl_principal": "{type user name test}",
+                            "acl_operation": "all",
+                            "acl_permission": "allow"
+                        }
+                    }, self.kafka_rpc_service_name), 1),
+            AbsoluteTestItem(
+                f'Delete group test',
+                lambda: self.execute_command_ignore_error(
+                    partial(self.rpk.group_delete, "test")),
+                partial(self.api_resource_match, "delete_groups", {
+                    "name": "test",
+                    "type": "group"
+                }, self.kafka_rpc_service_name), 1),
+            AbsoluteTestItem(
+                f'Alter Partition Reassignments',
+                lambda: self.execute_command_ignore_error(
+                    partial(alter_partition_reassignments_with_kcl, self.kcl,
+                            {topic_name: {
+                                1: [0]
+                            }})),
+                partial(self.api_resource_match,
+                        "alter_partition_reassignments", {
+                            "name": topic_name,
+                            "type": "topic"
+                        }, self.kafka_rpc_service_name), 1),
+            AbsoluteTestItem(
+                f'Alter Config (not-incremental)',
+                lambda: self.execute_command_ignore_error(
+                    partial(alter_config, self.default_client, {
+                        "log_message_timestamp_type": "CreateTime"
+                    }, False)),
+                partial(self.api_match, "alter_configs",
+                        self.kafka_rpc_service_name), 1),
+            AbsoluteTestItem(
+                f'Incremental Alter Config',
+                lambda: self.execute_command_ignore_error(
+                    partial(alter_config, self.default_client, {
+                        "log_message_timestamp_type": "CreateTime"
+                    }, True)),
+                partial(self.api_match, "incremental_alter_configs",
+                        self.kafka_rpc_service_name), 1),
+            AbsoluteTestItem(
+                f'List ACLs (no item)', lambda: self.rpk.acl_list(),
+                partial(self.api_match, "list_acls",
+                        self.kafka_rpc_service_name), 0)
+        ]
+
+        # Enable management now
+        self.logger.debug("Modifying event types")
+        self.modify_audit_event_types(['management'])
+
+        for test in tests:
+            self.logger.info(f'Running test "{test.name}"')
+            test.generate_function()
+            records = self.find_matching_record(test.filter_function,
+                                                test.valid_count, test.desc())
+            [validate_audit_schema(record) for record in records]
+
+        self.logger.debug("Disabling management")
+        self.modify_audit_event_types([])
+
+        for test in tests:
```

Comment:
why run the same logic after disabling management?

Reply:
Kind of duplicating the admin API test behavior (check that new messages don't show up).  But I probably don't need to do that.